spec.html

```diff
@@ -43421,6 +43421,7 @@ <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</
   <p><emu-xref href="#sec-properties-of-the-regexp-prototype-object"></emu-xref> In ECMAScript 2015, *"source"*, *"global"*, *"ignoreCase"*, and *"multiline"* are accessor properties defined on the RegExp prototype object. In previous editions they were data properties defined on RegExp instances.</p>
   <p><emu-xref href="#sec-atomics.notify"></emu-xref>: In ECMAScript 2019, `Atomics.wake` has been renamed to `Atomics.notify` to prevent confusion with `Atomics.wait`.</p>
   <p><emu-xref href="#sec-asyncfromsynciteratorcontinuation"></emu-xref>, <emu-xref href="#sec-asyncgeneratorresumenext"></emu-xref>: In ECMAScript 2019, the number of Jobs enqueued by `await` was reduced, which could create an observable difference in resolution order between a `then()` call and an `await` expression.</p>
+  <p><emu-xref href="#sec-runtime-semantics-unicodematchproperty-p"></emu-xref>, <emu-xref href="#sec-runtime-semantics-unicodematchpropertyvalue-p-v"></emu-xref>: In ECMAScript 2021, additional Unicode properties, property aliases, property values, and property value aliases are supported in Unicode property escapes within regular expressions, following changes in the upstream Unicode Standard.</p>
```

Comment:
I'm not sure how valuable this note really is, given that we'll likely be repeating it every edition.

Reply:
I agree. I think people expect Unicode releases to add new code points, new scripts, new properties, etc. which are reflected in a number of ways throughout the language. It's not exactly notable. Also, since we have a normative reference to the latest Unicode version, we can't even say this is a change across ECMA262 releases.

Reply:
@mathiasbynens We talked about this in the editor call today, and have decided that we only wish to include this information for significant changes like whitespace additions/removals or ID_Start/ID_Part removals, which this is not. Please remove this addition.

Reply:
Sorry for the churn! My concern is that I can review Annex E and know what i have to fix in my packages :-)

Reply:
Done.